src/frontend/components/SøknadsSteg/EøsSteg/Barn/Omsorgsperson.tsx

```diff
@@ -120,12 +121,24 @@ const Omsorgsperson: React.FC<OmsorgspersonProps> = ({ skjema, barn, periodeFunk
                     label={plainTekst(eøsForBarnTekster.idNummerOmsorgsperson.checkboxLabel)}
                 />
             </>
-            <SkjemaFeltInput
-                felt={skjema.felter.omsorgspersonAdresse}
-                visFeilmeldinger={skjema.visFeilmeldinger}
-                label={<TekstBlock block={eøsForBarnTekster.hvorBorOmsorgsperson.sporsmal} />}
-                description={plainTekst(eøsForBarnTekster.hvorBorOmsorgsperson.beskrivelse)}
-            />
+            <KomponentGruppe>
+                <>
```

Comment:
Er det nødvendig med tomme html-tagger rundt `SkjemaFeltInput` og `SkjemaCheckbox`?

Reply:
Hmm, ja på en måte, fordi den KomponentGruppe lager space mellom hvert element, men det skal ikke være det spacet mellom input og check fordi dem hører sammen. Men man kan spørre seg om man bør bruke noe annet enn KomponentGruppe her for å få resten av stylingen som KomponentGruppe har 🤔